src/platform/elements/form/Control.ts

```diff
@@ -234,372 +136,396 @@ export class NovoCustomControlContainerElement {
+                            });
+                            break;
+                        case 'focus':
+                            this.valueChangeSubscription = this.onFocus.debounceTime(300).subscribe(() => {
+                                this.executeInteraction(interaction);
+                            });
+                            break;
+                        case 'change':
+                            this.valueChangeSubscription = this.form.controls[this.control.key].valueChanges.debounceTime(300).subscribe(() => {
+                                this.executeInteraction(interaction);
+                            });
+                            break;
+                        case 'init':
+                            interaction.invokeOnInit = true;
+                            break;
+                        default:
+                            break;
+                    }
+                    if (interaction.invokeOnInit) {
+                        this.executeInteraction(interaction);
+                    }
+                }
+            }
+        }
+        this.templateContext = {
+            $implicit: this.form.controls[this.control.key],
+            methods: {
+                restrictKeys: this.restrictKeys.bind(this),
+                emitChange: this.emitChange.bind(this),
+                handleFocus: this.handleFocus.bind(this),
+                handlePercentChange: this.handlePercentChange.bind(this),
+                handleBlur: this.handleBlur.bind(this),
+                handleTextAreaInput: this.handleTextAreaInput.bind(this),
+                handleEdit: this.handleEdit.bind(this),
+                handleSave: this.handleSave.bind(this),
+                handleDelete: this.handleDelete.bind(this),
+                handleUpload: this.handleUpload.bind(this),
+                modelChange: this.modelChange.bind(this),
+                modelChangeWithRaw: this.modelChangeWithRaw.bind(this),
+                handleAddressChange: this.handleAddressChange.bind(this),
+                handleTyping: this.handleTyping.bind(this),
+            },
+            form: this.form,
         }
-      });
-    }
-  }
-
-  executeInteraction(interaction) {
-    if (interaction.script && Helpers.isFunction(interaction.script)) {
-      setTimeout(() => {
-        this.fieldInteractionApi.form = this.form;
-        this.fieldInteractionApi.currentKey = this.control.key;
-        try {
-          interaction.script(this.fieldInteractionApi, this.control.key);
-        } catch (err) {
-          console.info('Field Interaction Error!', this.control.key); // tslint:disable-line
-          console.error(err); // tslint:disable-line
+        this.templateContext.$implicit.tooltipPosition = this.tooltipPosition;
+        this.templateContext.$implicit.tooltip = this.tooltip;
+        this.templateContext.$implicit.startupFocus = this.control.startupFocus;//what does this do????
```

Comment:
This should focus the first field when the form is loaded, should see it on the form pages and should be autofocused